doc/rados/operations/health-checks.rst

```diff
@@ -315,12 +316,12 @@ The OSD can be removed from the CRUSH hierarchy with:
 OSD_OUT_OF_ORDER_FULL
 _____________________
 
-The utilization thresholds for `nearfull`, `backfillfull`, `full`,
-and/or `failsafe_full` are not ascending.  In particular, we expect
-`nearfull < backfillfull`, `backfillfull < full`, and `full <
+The utilization thresholds for `nearfull`, `backfillfull`, `full`, and/or
+`failsafe_full` are not ascending. In particular, the following pattern is
```

Comment:
Joe Cepher might not grok what "ascending" means there.  I want to sub "monotonically increasing" but I won't swear under oath that that is ideally accessible either.  But I think the following sentence explains it well enough to use the latter.